File: Source/client-model/src/main/java/model/intention/AbortAttackIntention.java
package model.intention;

import root.Point2D;
import root.model.event.ClientIntention;

public class AbortAttackIntention extends ClientIntention {

	public Point2D position;

	public AbortAttackIntention(Point2D position, String playerName) {
		super(ClientIntentionType.AbortAttack, playerName);

		this.position = position;
	}

}

File: Source/server/rabbit-game/src/SharedModel/ClientIntentions/AbortAttackIntention..cs

namespace RabbitGameServer.SharedModel.ClientIntentions
{
	public class AbortAttackIntention : ClientIntention
	{
		public Point2D position;

		public AbortAttackIntention(string playerName, Point2D position)
			: base(ClientIntentionType.AbortAttack, playerName)
		{
			this.position = position;
		}
	}
}